packages/dashmate/src/commands/doctor.js

```diff
@@ -0,0 +1,149 @@
+import process from 'process';
+import ConfigBaseCommand from '../oclif/command/ConfigBaseCommand.js';
+import getMetrics from '../util/getMetrics.js';
+import Report from '../doctor/report.js';
+
+export default class DoctorCommand extends ConfigBaseCommand {
+  static description = 'Generate a report about masternode';
+
+  static flags = {
+    ...ConfigBaseCommand.flags,
+  };
+
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param createRpcClient
+   * @param {DockerCompose} dockerCompose
+   * @param {getConnectionHost} getConnectionHost
+   * @param {Config} config
+   * @param createTenderdashRpcClient
+   * @param getServiceList
+   * @param getOperatingSystemInfo
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    args,
+    flags,
+    createRpcClient,
+    dockerCompose,
+    getConnectionHost,
+    config,
+    createTenderdashRpcClient,
+    getServiceList,
+    getOperatingSystemInfo,
+  ) {
+    const rpcClient = createRpcClient({
+      port: config.get('core.rpc.port'),
+      user: 'dashmate',
+      pass: config.get('core.rpc.users.dashmate.password'),
+      host: await getConnectionHost(config, 'core', 'core.rpc.host'),
+    });
+
+    const tenderdashRPCClient = createTenderdashRpcClient({
+      host: config.get('platform.drive.tenderdash.rpc.host'),
+      port: config.get('platform.drive.tenderdash.rpc.port'),
+    });
+
+    const report = new Report();
+
+    // OS INFO
+    console.log('Collecting Operating System Info');
+    const osInfo = await getOperatingSystemInfo();
```

Comment:
we want also

cpu: arch, manufacturer, brand
memory: free, available
osInfo: platform
load: avgLoad, currentLoad
disksIO: rIO, wIO
OpenFiles: max, allocated, available
si.inetLatency